src/java/fr/paris/lutece/plugins/appointment/web/AppointmentJspBean.java

```diff
@@ -557,8 +557,14 @@ public synchronized String getManageAppointments( HttpServletRequest request ) t
                     _filter.setEndingTimeOfSearch( endingDateTime.toLocalTime( ).toString( ) );
                 }
             }
-        List<AppointmentDTO> listAppointmentsDTO = AppointmentService.findListAppointmentsDTOByFilter( _filter );
-
+        _strCurrentPageIndex = AbstractPaginator.getPageIndex( request, AbstractPaginator.PARAMETER_PAGE_INDEX, _strCurrentPageIndex );
+        if ( _strCurrentPageIndex == null )
+        {
+            _strCurrentPageIndex = DEFAULT_CURRENT_PAGE;
+        }
+        _nItemsPerPage = AbstractPaginator.getItemsPerPage( request, AbstractPaginator.PARAMETER_ITEMS_PER_PAGE, _nItemsPerPage, _nDefaultItemsPerPage );
+        List<Integer> listAppointmentsIds = AppointmentService.findListAppointmentsIdsByFilter( _filter );
```

Comment:
This instruction should be called only in the case where _strCurrentPageIndex is null or equal to DEFAULT_CURRENT_PAGE

Reply:
If I refresh the list only when the table is first displayed (_strCurrentPageIndex is null or equal to DEFAULT_CURRENT_PAGE) I would not be able to display an appointment created after the table was displayed

Reply:
The request is expensive when called on each page. If the user wants to list the appointments created during pagination, they simply need to refresh the filter or go back to the first page.